src/game/StoryExplorerGame.java

```diff
@@ -91,6 +91,25 @@ public int getNumSatellitesToShow()
 		return textList;
 	}
 	
+	public ArrayList<String> getSatellitesTeaserImage()
+	{
+		ArrayList<String> imageList = new ArrayList<String>();
+		
+		for (StoryNode s : m_currentSatellites)
+		{
+			if (s.getTeaserImage() == null) {
```

Comment:
Not a big deal but if you want to keep with the current style, I've been putting braces on their own line.